docs/installing/manual_install.md

```diff
@@ -411,6 +411,13 @@ site in action.
 
 You can daemonize the process by starting it with the `--daemonize` option.
 
+
+If you anticipate getting a lot of traffic, you can configure thin to run as a cluster
```

Comment:
I think think we should generate the daemon through the rake task as suggested https://github.com/mysociety/alaveteli/issues/2952#issuecomment-169300575, but without dynamically setting the number of servers.

I think setting a default of 1 is the best idea, so that the init file has the `--servers` flag in all the correct places.